.eslintrc.js

```diff
@@ -8,5 +8,6 @@ module.exports = {
     'jsx-a11y/label-has-associated-control': ["error", {
       assert: "either",
     }],
+    'import/no-cycle': 'off', 
```

Comment:
I think these changes are not good, cause you import files by cycle, and do not need to disable these eslint rules.